lib/App/DH.pm

```diff
@@ -261,15 +261,31 @@ sub cmd_upgrade {
   return;
 }
 
+=cmd version
+
+Report version of connection L</--connection_name>
```

Comment:
I've now written `schema_version` and `database_version` which are the method names from DBIC::DeploymentHandler. Logic is already implemented there. Wasn't aware of the difference before.